gocourse11/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"gocourse11/aquarium"
+	"gocourse11/filter"
+)
+
+func main() {
+
+	first := aquarium.New(aquarium.WithAnimal(aquarium.Cat),
+		aquarium.WithChemichals([]aquarium.Chemichals{
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N}),
+		aquarium.WithSaltLevel(5),
+		aquarium.WithSize(10))
+	RunFilter(first, "first")
+
+	second := aquarium.New(aquarium.WithAnimal(aquarium.Cat),
+		aquarium.WithChemichals([]aquarium.Chemichals{
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N}),
+		aquarium.WithSaltLevel(5),
+		aquarium.WithSize(10))
+	RunFilter(second, "second")
+
+	third := aquarium.New(aquarium.WithAnimal(aquarium.Cat),
+		aquarium.WithChemichals([]aquarium.Chemichals{
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N,
+			aquarium.N,
+			aquarium.Mg,
+			aquarium.Mg,
+			aquarium.N,
+			aquarium.Mg}),
+		aquarium.WithSaltLevel(3),
+		aquarium.WithSize(8))
+	RunFilter(third, "third")
+}
+
+func RunFilter(a *aquarium.Aquarium, name string) {
+	filterSystem := filter.System{}
+	polution := filterSystem.CalculatePollution(*a)
+	if polution <= 5 {
+		fmt.Printf("Aquarium %s is clean\n", name)
+	} else {
```

Comment:
Якщо додати `return`, то можна обійтися без `else`:
```suggestion
		fmt.Printf("Aquarium %s is clean\n", name)
		return
	}
```

Ось як тут радять:
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88